### DISCUSSION TITLE
Code with codespaces feedback

### DISCUSSION THREAD

**Discussion by Chummy21** [skills/.github#442] [Code with Codespaces]
Nice work! 🎉 You created a codespace with a custom image!

You can customize your codespace by adding VS code extensions, adding features, setting host requirements, and much more.

Let's customize some settings in the .devcontainer.json file!

⌨️ Activity: Add customizations to the devcontainer file
Navigate to the .devcontainer/devcontainer.json file.

Add the following customizations to the body of the file before the last }.

 ,
 // Add the IDs of extensions you want installed when the container is created.
 "customizations": {
     "vscode": {
         "extensions": [
             "GitHub.copilot"
         ]
     },
     "codespaces": {
         "openFiles": [
             "codespace.md"
         ]
     }
 }
Click Commit changes and then select Commit changes directly to the main branch.

Create a new codespace by navigating to the landing page of your repository.

After following instructions for step three, 
on getting to  ( 4. Create a new codespace by navigating to the landing page of your repository.) it says {maximum running codespaces reached}
you have too many running code spaces. You will need to stop some before you start a new codespace. I couldn't continue from here. Please, kindly see to this.
Thank you.

**Comment by geekdaniels** [comment]
You can run just two codespaces at a time.

**Comment by trinneert** [comment]
As a relative beginner, the fact that each step required creating a new codespace, only to be followed by the warning that too many code spaces were running, made no sense whatsoever -- I have no idea what was being demonstrated.

**Comment by dlindsaymak** [comment]
I agree. I feel like I am just following instructions but am not internalizing what is actually going on. I'm coming from coding in R and I use it for data analysis, not programming - so I feel like I'm missing some vital information here.
